Makefile

```diff
@@ -186,6 +186,9 @@ bump-kubevirtci:
 fossa:
 	hack/dockerized "FOSSA_TOKEN_FILE=${FOSSA_TOKEN_FILE} ./hack/fossa.sh"
 
+format:
```

Comment:
would you mind calling it `fmt`?

Reply:
Why choose when you can have both?

Reply:
Sure, whatever you decide. Both work for me.

@zcahana is there a nice way of defining two rules for the same logic in Makefile syntax? Or you mean simply duplicating it?

If it requires duplication then I IMO we should stick with only one rule

Reply:
would be fine by me too.

Reply:
you can just add this:

```
fmt: format
```

Reply:
Cool! Learning something new every day :)

Done!